Sources/CoreLocationMiddleware/CoreLocationMiddleware.swift

```diff
@@ -43,25 +62,28 @@ public struct DeviceCapabilities: Equatable {
     public let isLocationServiceAvailable: Bool
 }
 
+// MARK: - REDUCERS
 
-let locationReducer = Reducer<LocationAction, LocationState> { action, state in
-    var state = state
-    switch action {
-    case .startMonitoring,
-         .stopMonitoring,
-         .requestAuthorizationStatus,
-         .requestAuthorizationType,
-         .requestPosition,
-         .requestDeviceCapabilities:
-        break
-    case let .gotAuthzStatus(status): state.authzStatus = status
-    case let .gotPosition(position): state.location = position
-    case .gotDeviceCapabilities, .receiveError : break
-    }
+extension Reducer where ActionType == LocationAction, StateType == LocationState {
+    static let location = Reducer<StatusAction, LocationState>.status.lift(action: \LocationAction.status)
+}
```

Comment:
Must be having a brain blip here... In a separate branch I commented out all the Reducer section. It's not being used. Not sure why we're using it in the Middleware. Can the Middleware's state be mutated ? I thought not...
But looking at some of the other Middleware implementations, I see some Reducers as well... like the [`MultiPeerAdvertiserMiddleware`](https://github.com/SwiftRex/MultipeerMiddleware/blob/master/Sources/MultipeerMiddleware/Advertiser/MultipeerAdvertiserReducer.swift).
@luizmb , I think I am missing something here...